prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -52,7 +52,10 @@ presubmits:
       containers:
       - image: alpine:3.15.0
         command:
-        - ./manifests/gcp_marketplace/test/presubmit.sh
+        - /bin/sh
+        args:
+        - -c
+        - sh ./manifests/gcp_marketplace/test/presubmit.sh
```

Comment:
If you want to stick with alpine, that's fine by me. Maybe`/bin/sh -c sh something.sh` can be simplified to`/bin/sh something.sh`?

Reply:
After trying the scripts, I found that I need to run more than one command, please see the updated commands.

Reply:
forgot to push?

Reply:
Sorry! I have pushed now.